core/domain/topic_domain.py

```diff
@@ -687,6 +687,25 @@ def delete_additional_story(self, story_id):
                 'The story_id %s is not present in the additional '
                 'story references list of the topic.' % story_id)
 
+    def prepublish_validate(self):
```

Comment:
(Just wanted to check!)
In the exp_domain we have a similar check within the validation method but covered under `strict` flag. If it's the same then I would prefer having a consistent pattern.

![image](https://user-images.githubusercontent.com/16653571/79082717-465e7680-7d46-11ea-9ea8-31a3bdfc93ab.png)

Reply:
Oh, ok, yeah changed it to use strict.